bin/node/testing/src/bench.rs

```diff
@@ -215,7 +256,10 @@ impl BenchDb {
 					function: Call::Balances(
 						BalancesCall::transfer(
 							pallet_indices::address::Address::Id(receiver),
-							1*DOLLARS
+							match block_type {
+								BlockType::RandomTransfers(_) => 1*DOLLARS,
+								BlockType::RandomTransfersReaping(_) => 99*DOLLARS,
```

Comment:
Why 99?

Reply:
1 fee+99 transfer = 100 (initial account endowment), so the balance should become 0